pkg/runtime/runtime.go

```diff
@@ -474,7 +480,6 @@ func (a *DaprRuntime) initRuntime(opts *runtimeOpts) error {
 
 	// Start proxy
 	a.initProxy()
-
```

Comment:
I suggest reverting these newline changes since I don't think they're contributing to this PR or to readability generally.